README.md

```diff
@@ -54,7 +53,7 @@ The list is a work in progress, please feel invited to [contribute](#contributin
 - [django-cms](https://docs.django-cms.org/) - django-cms's developer documentation is as extensive as it's well-organized. It uses the Furo theme. **#sphinx** **#large-project**.
 - [Ray](https://docs.ray.io/) - Ray is a documentation project spanning multiple software components. It uses several extensions from the Executable Book project. Features are showcased in [this twitter thread](https://twitter.com/readthedocs/status/1663923671470047234). **#sphinx** **#themes** **#large-project**.
 - [Scrapy](https://docs.scrapy.org/) - Embeds a lot of reference snippets and uses `sphinx-hoverxref` for quick reference tooltips. Lots of inspiration to be found in content organization. **#sphinx**.
-- [setuptools](https://setuptools.pypa.io/) - Lots of features, using the Furo theme. [Twitter thread](https://twitter.com/readthedocs/status/1546527820150718469) with some examples. **#sphinx** **#themes**.
+- [setuptools](https://setuptools.pypa.io/) - Lots of features, using the Furo theme. **#sphinx** **#themes**.
```

Comment:
This thread seems useful? No real reason not to keep it?

Reply:
Meh, it was giving 404 for some reason and the lint was complaining about that. That's why I removed it. It works now to me. Weird 🤷🏼